packages/rainbowkit/wallets/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "main": "../dist/wallets/walletConnectors/index.js"
```

Comment:
Similar to our `styles.css` alias, this is here mainly as backwards compatibility for Jest.